test/javascript/components/mentoring/session/FavoriteButton.test.js

```diff
@@ -19,12 +19,11 @@ test('mark student as a favorite', async () => {
   )
   server.listen()
 
-  render(
-    <FavoriteButton
-      isFavorite={false}
-      endpoint="https://exercism.test/favorites"
-    />
-  )
+  const student = {
```

Comment:
@kntsoriano This file was deleted. Was that intentional? If so, please re-delete it but ensure there's test coverage elsewhere. If it's not intentional, please fix it :)

Reply:
@iHiD This was intended to be deleted because we lifted state up to the parent component to keep all favorite buttons in sync :)